plugins/plugin_utils.h

```diff
@@ -9,4 +9,6 @@ void swap_store(void *dest, const void *pa, int size);
 int32_t deserialize_meta(uint8_t *smeta, int32_t smeta_len, int8_t *ndim, int64_t *shape,
                          int32_t *chunkshape, int32_t *blockshape);
 
+void index_unidim_to_multidim(uint8_t ndim, int64_t *shape, int64_t i, int64_t *index);
+
 void index_multidim_to_unidim(const int64_t *index, int8_t ndim, const int64_t *strides, int64_t *i);
```

Comment:
If we want `index_unidim_to_multidim ` and `index_unidim_to_multidim` to be public, I propose to rename them to `blosc2_unidim_to_multidim` and `blosc2_unidim_to_multidim` and add them to the end of blosc2.h

Reply:
It makes sense, I will put it on blosc2.h